### PR TITLE
Update Dockerfile.win10.min

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: Dockerfile.win10.min [hunk 1]
```diff
@@ -95,12 +92,11 @@ ARG CUDA_INSTALL_ROOT_WP="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v${
 
 WORKDIR /
 
-ARG TENSORRT_VERSION=8.5.1.7
-ARG TENSORRT_ZIP="TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-11.8.cudnn8.6.zip"
```
Comment (krishung5) [inline Dockerfile.win10.min:99]: Is `TENSORRT_ZIP` deleted by accident? I'm seeing `TENSORRT_ZIP` being used below.

**Review by krishung5 — CHANGES_REQUESTED** [review]

File: Dockerfile.win10.min [hunk 2]
```diff
@@ -66,10 +66,7 @@ RUN vcpkg.exe update
 RUN vcpkg.exe install openssl:x64-windows openssl-windows:x64-windows rapidjson:x64-windows re2:x64-windows boost-interprocess:x64-windows boost-stacktrace:x64-windows zlib:x64-windows pthread:x64-windows b64:x64-windows
 RUN vcpkg.exe integrate install
 
-# Install a recent version of cmake. The version installed with VS is
```
Comment (dyastremsky) [inline Dockerfile.win10.min:69]: We may want to keep this comment. Unless there is a reason to delete it.

Comment (mc-nv) [reply Dockerfile.win10.min:69]: I don't believe we need that comment anymore: 
1. ONNXRUNTIME requires 3.24.1 version as minimal as of 1.14.1 release 
2. Comment is irrelevant to me, CMAKE and VSCode are different frameworks where CMAKE actually is using VSCode compiler to build targets according to generated instructions.

File: Dockerfile.win10.min (hunk 1 above)
Comment (mc-nv) [reply Dockerfile.win10.min:99]: Yes, it was removed by mistake

**Review by dyastremsky — APPROVED** [review]

**Review by krishung5 — APPROVED** [review]

**Pull request merged**